Description: Do not emit a warning if the .haddock file is missing
 As it is quite common on Debian installations to install the -dev package
 without the -doc package.
Author: Joachim Breitner <nomeata@debian.org>

Index: ghc/utils/ghc-pkg/Main.hs
===================================================================
--- ghc.orig/utils/ghc-pkg/Main.hs	2016-02-08 11:31:08.897241576 +0100
+++ ghc/utils/ghc-pkg/Main.hs	2016-02-08 11:31:08.893241486 +0100
@@ -1585,8 +1585,10 @@
   mapM_ (checkDir True  "library-dirs") (libraryDirs pkg)
   mapM_ (checkDir True  "include-dirs") (includeDirs pkg)
   mapM_ (checkDir True  "framework-dirs") (frameworkDirs pkg)
-  mapM_ (checkFile   True "haddock-interfaces") (haddockInterfaces pkg)
-  mapM_ (checkDirURL True "haddock-html")       (haddockHTMLs pkg)
+  -- In Debian, it is quite normal that the package is installed without the
+  -- documentation. Do not print a warning there.
+  -- mapM_ (checkFile   True "haddock-interfaces") (haddockInterfaces pkg)
+  -- mapM_ (checkDirURL True "haddock-html")       (haddockHTMLs pkg)
   checkDuplicateModules pkg
   checkExposedModules db_stack pkg
   checkOtherModules pkg
